client/ava.config.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  files: [
+    'test/**/*.spec.js'
```

Comment:
Maybe add another pattern for in-browser testing or use `name.browser.spec.js` for distinction.